tests/putobject_test.py

```diff
@@ -151,6 +151,64 @@ def test_aws_put_in_subdirectory(self):
             os.remove(get_filename)
             assert_command(f'irm -rf {self.bucket_irods_path}/{put_directory}')
 
+    def test_mc_put_in_bucket_root_small_file(self):
```

Comment:
It's not clear what this test name is describing. Consider using more words or rephrasing.

Reply:
I reordered the name a bit.  It is now `test_mc_put_small_file_in_bucket_root`.  Is that better?

I could also change it to `test_mc_put_a_small_file_in_the_bucket_root` if that is better?

Also, is the `mc` part for `minio client` confusing?

Reply:
Those sound good to me.